local.tf

```diff
@@ -29,51 +29,54 @@ locals {
+    ami_id                            = ""                          # AMI ID for the eks linux based workers. If none is provided, Terraform will search for the latest version of their EKS optimized worker AMI based on platform.
+    ami_id_windows                    = ""                          # AMI ID for the eks windows based workers. If none is provided, Terraform will search for the latest version of their EKS optimized worker AMI based on platform.
+    asg_desired_capacity              = "1"                         # Desired worker capacity in the autoscaling group and changing its value will not affect the autoscaling group's desired capacity because the cluster-autoscaler manages up and down scaling of the nodes. Cluster-autoscaler add nodes when pods are in pending state and remove the nodes when they are not required by modifying the desirec_capacity of the autoscaling group. Although an issue exists in which if the value of the asg_min_size is changed it modifies the value of asg_desired_capacity.
+    asg_max_size                      = "3"                         # Maximum worker capacity in the autoscaling group.
+    asg_min_size                      = "1"                         # Minimum worker capacity in the autoscaling group. NOTE: Change in this paramater will affect the asg_desired_capacity, like changing its value to 2 will change asg_desired_capacity value to 2 but bringing back it to 1 will not affect the asg_desired_capacity.
+    asg_force_delete                  = false                       # Enable forced deletion for the autoscaling group.
+    asg_initial_lifecycle_hooks       = []                          # Initital lifecycle hook for the autoscaling group.
+    default_cooldown                  = null                        # The amount of time, in seconds, after a scaling activity completes before another scaling activity can start.
+    health_check_type                 = null                        # Controls how health checking is done. Valid values are "EC2" or "ELB".
+    health_check_grace_period         = null                        # Time in seconds after instance comes into service before checking health.
+    instance_type                     = "m4.large"                  # Size of the workers instances.
+    instance_store_virtual_name       = "ephemeral0"                # "virtual_name" of the instance store volume.
+    spot_price                        = ""                          # Cost of spot instance.
+    placement_tenancy                 = ""                          # The tenancy of the instance. Valid values are "default" or "dedicated".
+    root_volume_size                  = "100"                       # root volume size of workers instances.
+    root_volume_type                  = "gp3"                       # root volume type of workers instances, can be "standard", "gp3", "gp2", or "io1"
```

Comment:
was this change intended?

Reply:
Nope https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1404

Reply:
Just released [v17.0.1](https://github.com/terraform-aws-modules/terraform-aws-eks/releases/tag/v17.0.1).

Thank you very much @james-callahan